commands/setup.js

```diff
@@ -117,6 +117,15 @@ module.exports = {
 
     await interaction.deferReply();
     
+    const botMember = interaction.guild.members.me;
+    if (!channel.permissionsFor(botMember).has(PermissionsBitField.Flags.SendMessages)) {
+      await interaction.editReply({
+        content: "I do not have permission to send messages in the specified channel.",
```

Comment:
```suggestion
        content: "Je n'ai pas la permission d'envoyer de messages dans le salon indiqué.",
```
puisque le reste des messages est en français